source/extensions/stat_sinks/common/statsd/statsd.cc

```diff
@@ -100,6 +105,11 @@ TcpStatsdSink::TcpStatsdSink(const LocalInfo::LocalInfo& local_info,
   tls_->set([this](Event::Dispatcher& dispatcher) -> ThreadLocal::ThreadLocalObjectSharedPtr {
     return std::make_shared<TlsSink>(*this, dispatcher);
   });
+  if (_prefix.size() == 0) {
```

Comment:
`prefix.empty()`, initialize in init list per below.

Reply:
Changed initialization to init list for both UdpStatsdSink and TcpStatsdSink